department-of-veterans-affairs/caseflow
# CASEFLOW-1021 update edit NOD date failure message

Resolves [#15919 ](https://vajira.max.gov/browse/CASEFLOW-1021)

### Description
The Edit NOD Modal was updated with the following:

- Verify that an alert banner appears with the following wording: 
--  New NOD Date Not Saved
-- The NOD Date has not be edited because the new date affects the eligibility of one or more issues. See below for more information.
-  Verify that the decision date appears for each issue.
- Verify that the following message appears: 
--If you believe this information needs to be updated, please reach out to the Clerk of the Board.
- Verify each issue has a bullet point and not an ordered number
- Verify that there is no Cancel link
- Verify that the success banner no longer displays the word "timeliness" and instead displays "eligibility" (upon successful NOD date update)

### Acceptance Criteria

### Testing Plan

- Go to veteran file 152003980
- Click 'Edit NOD link' and change nod date to one week after original
- Select a change reason
- Click submit
- Verify that unaffected and unaffected issues appear in modal and replace the form in the modal
- Verify that the decision date appears in a line below each issue.
- Verify that there is a message below the issues telling the user to follow up with Clerk of the Board if changes desired.

### User Facing Changes

 BEFORE
<img width="365" alt="before_screenshot" src="https://user-images.githubusercontent.com/23639439/108569475-f2171200-72d9-11eb-9967-e1302554cfcc.png">

AFTER
<img width="250" alt="After_screenshot" src="https://user-images.githubusercontent.com/23639439/108569324-b1b79400-72d9-11eb-80d4-710345848676.png">
